src/SprykerEco/Zed/Adyen/Business/Writer/AdyenWriter.php

```diff
@@ -117,6 +129,20 @@ public function saveApiLog(
         });
     }
 
+    /**
+     * @param \Generated\Shared\Transfer\AdyenNotificationsTransfer $adyenNotificationsTransfer
+     *
+     * @return void
+     */
+    public function saveNotifications(AdyenNotificationsTransfer $adyenNotificationsTransfer): void
+    {
+        $this->getTransactionHandler()->handleTransaction(function () use ($adyenNotificationsTransfer) {
```

Comment:
I wonder if it'll work if you pass the `$adyenNotificationsTransfer` as a param to this callable? If so, it might be better to do so, because then you'll be able to declare the type.